src/aml_service/index.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const httpRequest = require('../request');
+const AmlResultClass = require('../request/aml.result').AmlResult;
+const {Payload} = require('../request/payload');
+const constants = require('../yoti_common/constants');
+const ON_PEP_LIST_ATTR = constants.ON_PEP_LIST_ATTR;
+const ON_FRAUD_LIST_ATTR = constants.ON_FRAUD_LIST_ATTR;
+const ON_WATCH_LIST_ATTR = constants.ON_WATCH_LIST_ATTR;
+
+const AmlResult = function(rawResult){
+  this.isOnPepList = rawResult[ON_PEP_LIST_ATTR];
```

Comment:
The way we are using constants here feels more confusing than helpful to me. Can I get some more opinions @echarrod @qzagarese @laurent-yoti 
I would just use `rawResult['on_pep_list'];`

Reply:
Can you just do `rawResult[constants.ON_FRAUD_LIST_ATTR];`, which removes the need for declaring the extra intermediary variables above, but also doesn't declare the string constant in the method

Reply:
Yes that could do, it just seemed a bit long to me hence I used the declaration above.

Reply:
As long as it's not confusing to any one else.